### ISSUE TITLE
Class 'Services_Twilio' not found

### ISSUE THREAD

**Issue by kevupton** [aloha/laravel-twilio#135]
It seems that my build is unable to find any of the services that your code is using, 
am I missing an installation step.

I later inspected twilio php and it seems these were back in legacy days, is it the fact that this is a legacy requirement?

Class 'Services_Twilio' not found

**Comment by hannesvdvreken** [comment]
@jmctwilio ^ there you have it. I guess I'll have to remove tag 3.0.1 and make a 4.0.0 version which adopts to your update namespacing scheme?

**Comment by LeVadim** [comment]
This plugin does not support Twilio`s latest SDK version yet it installs the latest version of it. Updating main composer.json `"twilio/sdk" : "4.12.0"` and removing/installing the twilio package in the vendor folder solves the issue.

**Comment by hannesvdvreken** [comment]
Run `composer require twilio/sdk:"^4.0"` to temporarily fix this.

**Comment by LeVadim** [comment]
`"twilio/sdk" : "4.12.0"` version is ok. Hopefully in the future this package would support latest SDK of Twilio - otherwise eventually it becomes deprecated.

**Comment by hannesvdvreken** [comment]
It will once I've found some time

**Comment by LeVadim** [comment]
Ok. Anyways - great work of providing ease of use of the core features of the Twilio API`s to most of the developers!

**Comment by phpspider** [comment]
How can i use 4.12.0 version with this repo? When updating composer it's fetching the 3.12.0 as per the dependency.

**Comment by hannesvdvreken** [comment]
@phpspider
``` 
composer require twilio/sdk:"^4.0"
composer require aloha/twilio:"^3.0"
```

**Comment by hannesvdvreken** [comment]
fixed with 3.0.2
